packages/fxa-settings/src/components/ConnectedServices/Service.tsx

```diff
@@ -2,36 +2,30 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import React from 'react';
+import React, { useState } from 'react';
 
 import { LinkExternal } from 'fxa-react/components/LinkExternal';
-import { useBooleanState } from 'fxa-react/lib/hooks';
-
-import { Checkbox } from '../Checkbox';
 import { DeviceLocation } from '../../models/Account';
 import { Icon } from './Icon';
-import { Modal } from '../Modal';
-import { useAlertBar } from '../../lib/hooks';
-import { VerifiedSessionGuard } from '../VerifiedSessionGuard';
 
 export function Service({
   name,
   deviceType,
   location,
   lastAccessTimeFormatted,
   canSignOut,
+  handleSignOut,
 }: {
   name: string;
   deviceType: string | null;
   location: DeviceLocation;
   lastAccessTimeFormatted: string;
   canSignOut: boolean;
+  handleSignOut: any;
```

Comment:
`() => void` instead of `any`?

Reply:
Is there a reason for the `any` here?

Reply:
Now that I've got TS-check re-enabled, your suggestion works :+1: